documentation/release_notes.markdown

```diff
@@ -14,11 +14,46 @@ canonical: "/puppetdb/latest/release_notes.html"
 [metrics]: ./api/metrics/v1/changes-from-puppetdb-v3.html
 [pqltutorial]: ./api/query/tutorial-pql.html
 
+4.1.2
+-----
+
+PuppetDB 4.1.2 is a backward-compatible bugfix release that speeds up node
+queries, fixes a bug that slowed garbage collection, and improves error logging.
+
+### Bug Fixes and Maintenance
+
+* Queries to the /nodes endpoint should be faster, due to the removal of the
+  index on certnames(latest_report_id).
```

Comment:
"Fixed a regression in the performance of the nodes endpoint by removing an index causing the PostgreSQL query planner to behave poorly." @wkalt thoughts?